src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,189 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { Children, cloneElement, FC, isValidElement, ReactNode, useEffect, useRef, useState } from "react";
+import { TabItemProps } from "@components/Tabs/TabItem";
+import { merge } from "@utilities/merge";
+import { IconMore } from "@foundation/Icon";
+import { Badge } from "@components/Badge";
+import { LayoutGroup, motion } from "framer-motion";
+import { Button, ButtonSize, ButtonStyle } from "@components/Button";
+import { Flyout } from "@components/Flyout";
+import { MenuItem } from "@components/MenuItem";
+
+export enum TabsPaddingX {
+    Small = "Small",
+    Medium = "Medium",
+    Large = "Large",
+}
+
+export enum TabSize {
+    Small = "Small",
+    Large = "Large",
+}
+
+export type TabsProps = {
+    layoutGroupId?: string;
```

Comment:
We can use `useMemoizedId` from `src/hooks/useMemoizedId.ts` instead of having to pass it

Reply:
Nice! So much better...I'm updating. Thanks.